### DISCUSSION TITLE
How to desgin my own plugins like gauge in open MCT.

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#3845] [Community Q&A]
Please tell how to desgin my own plugins like gauge in open MCT.  Whole frame work of Open MCT is changed in old framework documentation it was told how to desgin todo plugin. Similarly is there any documentation for present version.

**Comment by jvigliotta** [comment]
hi @Vaibhav2121 I'm going to move this over to discussions as it's more suitable there.

**Comment by jvigliotta** [comment]
Hey @Vaibhav2121 we do have our openmct-tutorial repo that shows you how to develop a simple plugin: https://github.com/nasa/openmct-tutorial#step-1---defining-a-new-plugin and we also have documentation in our main repo as well: https://github.com/nasa/openmct/blob/master/API.md#defining-and-installing-a-new-plugin
